webapp/src/pages/EditChatroomDetailsPage/EditChatroomDetailsPage.tsx

```diff
@@ -0,0 +1,217 @@
+    password: '',
+    confirmationPassword: '',
+    isPublic: isPublic,
+  };
+  const [formValues, setFormValues] =
+    useState<UpdateChatroomDto>(initialFormValues);
+
+  useEffect(() => {
+    if (chatroom) {
+      setIsPublic(chatroom.isPublic);
+    }
+  }, [chatroom]);
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    setFormValues((previousValues: any) => {
+      return { ...previousValues, [name]: value };
+    });
+  };
+
+  function hasValidFormValues() {
+    if (formValues.password !== formValues.confirmationPassword) {
+      warn('Passwords are different');
+      return false;
+    }
+    return true;
+  }
+
+  async function updateChatroomDetails() {
+    if (!hasValidFormValues()) {
+      return;
+    }
+    try {
+      await chatApi.chatControllerUpdateChatroom({
+        chatroomId: chatroomId!,
+        updateChatroomDto: {
+          name: formValues.name ? formValues.name : initialFormValues.name,
+          oldPassword: formValues.oldPassword || null,
+          password: formValues.password || null,
+          confirmationPassword: formValues.confirmationPassword || null,
+          isPublic: isPublic,
+        },
+      });
+      notify('Chatroom details successfully updated');
+      navigate(CHAT_URL);
+    } catch (error) {
+      let errMessage = '';
+      if (error instanceof ResponseError) {
+        errMessage = `${error.response.statusText}`;
+      } else if (error instanceof Error) {
+        errMessage = `${error.message}`;
+      }
+      warn(errMessage);
+    }
+  }
+  const handleOnSubmit = (e: React.FormEvent<HTMLFormElement>): void => {
+    e.preventDefault();
+    updateChatroomDetails().catch((e) => console.error(e));
+  };
+  const deleteChatoom = () => {
+    if (window.confirm('Are you sure you want to delete the chatroom?')) {
+      try {
+        chatApi.chatControllerDeleteChatroom({
+          chatroomId: chatroomId!,
+        });
+        notify('Chatroom successfully deleted');
+        navigate(CHAT_URL);
+      } catch (error) {
+        let errMessage = '';
+        if (error instanceof ResponseError) {
+          errMessage = `${error.response.statusText}`;
+        } else if (error instanceof Error) {
+          errMessage = `${error.message}`;
+        }
+        warn(errMessage);
+      }
+    }
+  };
+
+  if (!chatroom) {
+    return (
+      <div className="edit-chatroom-details">
+        <div className="edit-chatroom-details-loading">
+          <Loading />
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <div className="edit-chatroom-details">
+      <Header icon={IconVariant.ARROW_BACK} onClick={goBack}>
+        {`edit/${chatroom.name}`}
+      </Header>
+      <div className="edit-chatroom-details-avatar-properties">
+        <div className="edit-chatroom-details-avatar">
+          <LargeAvatar
+            url={WILDCARD_AVATAR_URL}
+            editUrl={EDIT_AVATAR_URL}
```

Comment:
EDIT_AVATAR_URL should be probably specified. The one we had until now was EDIT_USER_AVATAR_URL maybe this one is EDIT_CHATROOM_AVATAR_URL ? Or maybe we want a common module for all avatars with a different url for bot user and chatroom interfaces?
Maybe just leave a TODO, because right now it's just linking to edit the authUser avatar, which is  not what it's supposed to do.

Reply:
I will leave a TODO and create an issue about it. TBD